qcengine/programs/nwchem/runner.py

```diff
@@ -238,7 +238,7 @@ def parse_output(
             qcvars["CURRENT HESSIAN"] = nwhess
 
         # Normalize the output as a float or list of floats
-        retres = qcvars[f"CURRENT {input_model.driver.upper()}"]
+        retres = qcvars[f"CURRENT ENERGY" ]
```

Comment:
as-is this will mess up gradient and hessian jobs. want the original line except for properties driver

Reply:
Okay I pushed the change.  I added an if statement for the properties driver